### DISCUSSION TITLE
Beginner steps: trouble running sample project

### DISCUSSION THREAD

**Discussion by fbcosentino** [mcu-tools/mcuboot#1718] [Q&A]
I hope I won't embarrass myself with this question, but I'm having trouble getting the sample project to run.

I'm using Zephyr and a nRF52DK (52832). I have Zephyr SDK and toolchain working fine and tested. 
When I compile `mcuboot\boot\zephyr` I get no compilation errors and all is fine, then I flash into the board and I see the first serial output line at boot:
`*** Booting Zephyr OS build zephyr-v3.3.0-4282-g56f66334b050 ***`

and... _nothing else_.

I then compile the `hello-world`, flash it, and same thing. Only that single line is printed.

If I remove the `CONFIG_BOOTLOADER_MCUBOOT=y` from `hello-world` making it run without mcuboot, I get it to run and print just fine, which confirms my Zephyr compiler+sdk toolchain and flashing interface are working fine as well as the hardware.

I expected the bootloader + `hello-world` sample project to need no manual configuration from my part at all, as that is the purpose of the `hello-world` starting project anyway. Or did I assume wrong?

Am I doing something shamefully wrong?

**Comment by nordicjm** [comment]
The samples in mcuboot should not be used, it's fine to build the bootloader but for samples, build those directly in zephyr. There was an issue in 3.3 whereby the default logging level was changed in zephyr, and this removed logging from mcuboot, this is fixed in 3.4. I would recommend getting zephyr 3.4 (or main) and building an application (e.g. hello_world) using sysbuild - if you enable mcuboot in the sysbuild configuration, this will build both the application and mcuboot and you can flash them to a board with one `west flash` command
